using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -204,173 +281,331 @@ ACPTarget.getTntId({tntId in
+let mboxParameters = [
+"membership": "prime"
+]
+
+// Product parameters
+let productParameters = [
+"id": "CEDFJC",
+"categoryId": "Electronics"
+]
+
+// Order parameters
+let orderParameters = [
+"id": "NJJICK",
+"total": "650",
+"purchasedProductIds": "81, 123, 190"
+]
+
+// Profile parameters
+let profileParameters = [
+"ageGroup": "20-32"
+]
+
+// Create Target parameters
+let product = ACPTargetProduct(id: "24D334", categoryId: "Stationary")
+let order = ACPTargetOrder(id: "ADCKKBC", total: NSNumber(value: 400.50), purchasedProductIds: ["34", "125"])
+let targetParameters = ACPTargetParameters(parameters: nil, profileParameters: nil, product: product, order: order)
+
+ACPTarget.locationClicked(withName: "cartLocation", targetParameters: targetParameters)
 ```
 {% endtab %}
 
 {% tab title="React Native" %}
-### resetExperience
+
+#### Syntax
+
+```javascript
+locationClickedWithName(name: string, parameters?: ACPTargetParameters)
+```
+
+* _name_ is a string containing mbox location for which click notification is to be sent to Target.
+* _parameters_ is the configured `ACPTargetParameters` for the request.
+
+#### Examples
 
 **JavaScript**
 
 ```javascript
-ACPTarget.resetExperience();
+// Mbox parameters
+var mboxParameters = {"membership": "prime"};
+
+// Product parameters
+var productParameters = new ACPTargetProduct("CEDFJC", "Electronics");
+
+// Order parameters
+var orderParameters = new ACPTargetOrder("NJJICK", 650, ["81","123","190"]);
+
+// Profile parameters
+var profileParameters = {"ageGroup": "20-32"};
+
+// Create Target parameters
+var product = new ACPTargetProduct("24D334", "Stationary");
+var order = new ACPTargetOrder("ADCKKBC", 400.50, ["34","125"]);
+var targetParameters = new ACPTargetParameters(null, null, product, order);
+
+ACPTarget.locationClickedWithName("cartLocation", targetParameters);
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## Set preview restart deeplink
+## locationsDisplayed
 
-Use this API to set the Target preview URL to be displayed when the preview mode is enabled and preview selections are confirmed.
+Use this API to send a location \(mbox\) display notification to configured Target server. This API should be invoked for a prefetched mbox, after the mbox content is retrieved using [retrieveLocationContent API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#retrieve-location-content-requests). If no previous prefetch request is made and the mbox content is retrieved using `retrieveLocationContent` API, then calling this API does not trigger a notification request to the Target server.
+
+{% hint style="warning" %}
+Do not use this API with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#load-target-requests). For a prefetched mbox, on calling the deprecated `loadRequests` API, mbox display notification is internally sent to the Target server by the SDK.
```

Comment:
Please reword as follows:
"Do not use this API with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#load-target-requests) API. For a prefetched mbox, after calling the deprecated `loadRequests` API, the mbox display notification is sent internally to the Target server by the SDK."